internal/pkg/cli/local_run.go

```diff
@@ -56,15 +119,182 @@ func (o *localRunOpts) Validate() error {
 
 // Ask prompts the user for any unprovided required fields and validates them.
 func (o *localRunOpts) Ask() error {
-	//TODO(varun359): Validate and Ask SvcEnvName
-
+	if err := o.validateOrAskWorkloadName(); err != nil {
+		return err
+	}
+	if err := o.validateOrAskEnvName(); err != nil {
+		return err
+	}
 	return nil
 }
 
 // Execute builds and runs the workload images locally.
 func (o *localRunOpts) Execute() error {
-	//TODO(varun359): Get build information from the manifest and task definition for workloads
+	taskDef, err := o.ecsLocalClient.TaskDefinition(o.appName, o.envName, o.wkldName)
+	if err != nil {
+		return fmt.Errorf("get task definition: %w", err)
+	}
+
+	_, err = o.getDecryptedSSMSecrets(taskDef, o.sess)
+	if err != nil {
+		return err
+	}
+
+	_, err = o.getDecryptedSecretManagerSecrets(taskDef, o.sess)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (o *localRunOpts) getDecryptedSSMSecrets(taskDef *awsecs.TaskDefinition, awsSession *session.Session) ([]decryptedSecret, error) {
```

Comment:
I feel like this method shouldn't be in this file because it's not specific to local run. Can we move it to the `ecs` pkg (not the aws ecs one)? 

```go
func DecryptedSSMSecrets(secrets, secretGetter) ([]decryptedSecret, error)
```